website/content/configuration/troubleshooting.md

```diff
@@ -5,14 +5,17 @@ weight: 20
 
 Once you setup a loadbalancer on Kubernetes and it doesn't _seem_ to work right away, here are a few tips to troubleshoot.
 
-SSH into one or more of your nodes and use `arping` and `tcpdump` to verify the ARP requests pass through your network.
+- Check that the LoadBalancer service has endpoints (`kubectl get endpoints`) that are not pending - if they are, MetalLB will not respond to ARP requests for that service's external IP
```

Comment:
That command will probably run fine on most clusters, but just not for the desired service. Can you make it obvious that the service (and maybe ns too) needs to be specified? Like `kubectl -n <namespace> get endpoints <service>`?